apps/collabcal/src/redux/store.ts

```diff
@@ -57,6 +60,10 @@ export const account = {
 export const selectAccount = (state: RootState) => state.account;
 export const useIsAuthed = () => useSelector(selectAccount).session !== null;
 
+export const userData = userDataSlice.actions;
+export const selectUserData = (state: RootState) => state.userData;
+
+// PERF: make this an action to allow turning on/off?
 repos.appointments.stream().subscribe((appointment) => {
   store.dispatch(
     appointmentsSlice.actions.setOne(sanitiseAppointment(appointment))
```

Comment:
Seeing a lot of spam from this in the console. We should probably only start this once _after_ fetching all the appointments in a single query.